documentation/modules/about-hooks-for-migration-plans.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="about-hooks-for-migration-plans_{context}"]
+= About hooks for {project-short} migration plans
+
+You can add hooks to {project-first} migration plans using either the {project-short} CLI or the {project-short} user interface, which is located in the {ocp} web console.
+
+* Hooks that perform operations on VMs before migration are called _pre-migration_ hooks.  A pre-migration hook is performed on a VM that is located on the provider and prepares the VM for migration.
```

Comment:
As always, up to you what you want to incorporate. :-)

Pre-migration hooks are hooks that perform operations on a VM that is located on a provider, which prepares the VM for migration.